docs/developer-guide.md

```diff
@@ -39,34 +61,3 @@ In order to avoid potential side effects using `lerna version/publish` commands,
    ```sh
        yarn sync-version --name=<package-name>
    ```
-
-# Setup
-
-- Clone this repository
-- Run `yarn` to install package dependencies
-- Run `yarn build` to compile all packages
-- Run `yarn start` in an example app to test
-
-```sh
-yarn bootstrap
-yarn llink
-yarn build
-```
```

Comment:
Couldn't figure out how these two lists of commands were supposed to be used together. I changed it to the instructions Kristen gave me in Notion